data-tool/Makefile

```diff
@@ -36,102 +46,56 @@ clean-test: ## clean test files
 	rm -f .coverage
 	rm -fr htmlcov/
 
+install-all: clean install-prefect install-sqlalchemy install ## Install all virtual environments
+
+install-prefect: ## Install prefect server environment
+	test -f $(PREFECT_VENV_DIR)/bin/activate || $(PYTHON_VERSION) -m venv $(PREFECT_VENV_DIR) ;\
+	. $(PREFECT_VENV_DIR)/bin/activate ;\
+	pip install --upgrade pip ;\
+	pip install -r requirements/prefect.txt
+
+install-sqlalchemy: ## Install SQLAlchemy 1.4.44 environment
+	test -f $(SQLALCHEMY_VENV_DIR)/bin/activate || $(PYTHON_VERSION) -m venv $(SQLALCHEMY_VENV_DIR) ;\
+	. $(SQLALCHEMY_VENV_DIR)/bin/activate ;\
+	pip install --upgrade pip ;\
+	pip install "SQLAlchemy==1.4.44"
+
+install: clean ## Install main application environment
+	test -f $(VENV_DIR)/bin/activate || $(PYTHON_VERSION) -m venv $(VENV_DIR) ;\
+	. $(VENV_DIR)/bin/activate ;\
+	pip install --upgrade pip ;\
+	pip install -r requirements.txt ;\
+	pip install --no-deps git+https://github.com/bcgov/lear.git#egg=legal_api\&subdirectory=legal-api ;\
+	pip install --no-deps git+https://github.com/bcgov/business-schemas.git@2.16.1#egg=registry_schemas ;\
+	pip install --no-deps git+https://github.com/bcgov/lear.git#egg=sql-versioning\&subdirectory=python/common/sql-versioning
```

Comment:
lots of conflicts so pulled in these 3 with `--no-deps` to allow for more control of library versions in `requirements.txt`